src/redux/home/homeSlice.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-param-reassign */
```

Comment:
- [Optional] We could have tried to resolve the error flagged by Linters to make sure that our codebase is up to standard :+1: .